notifications_utils/logging.py

```diff
@@ -160,46 +141,9 @@ class RequestIdFilter(logging.Filter):
     def request_id(self):
         if has_request_context() and hasattr(request, 'request_id'):
             return request.request_id
-        else:
-            return 'no-request-id'
 
-    def filter(self, record):
-        record.request_id = self.request_id
-
-        return record
+        return 'no-request-id'
```

Comment:
return request.request_id if has_request_context() and hasattr(request, 'request_id') else 'no-request-id'

Reply:
I dig it, but that will give me a Flask8 error about the line being too long.  Then I would have to back-slash it, and that less readable that what's already there.

Reply:
Isn't flake8 120 columns?

Reply:
Yeah, found it:
![image](https://user-images.githubusercontent.com/16893311/236563649-23341c50-c435-46b8-aec5-0224c93f3c54.png)

Reply:
That sounds correct, but this is 124 columns with the preceding indentation.